app/views/aleph/full_item_status.html.erb

```diff
@@ -2,49 +2,68 @@
 <ul class="list-local-locations">
 <% @status.each do |s| %>
   <li class="result-local-location">
-    <%= s[:library] %> <%= s[:collection] %>: <%= s[:call_number] %>
-    <% if s[:description].present? %> <%= s[:description] %> <% end %>
-    <span class="sr"><%= s[:label] %></span>
+    <div class="location-wrap">
+      <span class="location">
+        <span class="library"><%= s[:library] %></span>
+        <span class="collection"><%= s[:collection] %>:</span>
+        <span class="callno"><%= s[:call_number] %></span>
+        <% if s[:description].present? %> <%= s[:description] %> <% end %> <!-- unknown -->
```

Comment:
What are the unknown flags for?

Reply:
oh right - i meant to ask about those. Nothing shows in the markup so i don't know what those do.